### ISSUE TITLE
At the beginning of FY24, compare the FIT and FIMS strategic plans and determine needs to edit or add tasks to align the two projects

### ISSUE THREAD

**Issue by k-doering-NOAA** [noaa-fisheries-integrated-toolbox/FIT-committee#54]

**Comment by ChristineStawitz-NOAA** [comment]
Yes! After that I plan to update our strategic plan with the prioritized items - we are behind the times for FY24

**Comment by k-doering-NOAA** [comment]
@ChristineStawitz-NOAA and @k-doering-NOAA discussed this on 4/30. We agreed that right now there is little overlap between FIT and FIMS work. The only action we felt necessary was to onboard FIMS to the FIT, which has been completed! See the [FIMS page](https://noaa-fisheries-integrated-toolbox.github.io/FIMS).
